presences/src/main/resources/sql/scriptSQLReverse/044-MA-967-count-alert-by-line.sql

```diff
@@ -0,0 +1,13 @@
+-- Delete old data due to the modification of the table
+DELETE FROM presences.alerts WHERE 1 = 1;
+DELETE FROM presences.alert_history WHERE 1 = 1;
```

Comment:
Idem pour truncate/delete sans condition

Reply:
DELETE FROM presences.alerts = Unsafe query: 'Delete' statement without 'where' clears all data in the table
TRUNCATE TABLE presences.alerts ☑